docs/sources/whatsnew/whats-new-in-v9-0.md

```diff
@@ -215,6 +215,25 @@ In the InfluxDB data source, browser access mode was deprecated in grafana 8.0.0
 
 The rename by regex transformation has been improved to allow global patterns of the form `/<stringToReplace>/g`. Depending on the regex match used, this may cause some transformations to behave slightly differently. You can guarantee the same behavior as before by wrapping the match string in forward slashes (/), for example, (._) would become /(._)/. ([Github Issue #48179](https://github.com/grafana/grafana/pull/48179))
 
+### Clock Panel
+
+We have updated [clock panel](https://grafana.com/grafana/plugins/grafana-clock-panel/) to version `2.0.0` and encourage everyone to update as soon as possible. It will cause the Grafana to [crash](https://github.com/grafana/clock-panel/issues/106) when being used in a dashboard.
```

Comment:
Get the impression that v2.0.0 will cause grafana to crash. Maybe something along the lines "Versions before 2.0.0 could crash on grafana v9 and above"

Reply:
I agree. Right now this text is a bit misleading as v2.0.0 won't work in Grafana 7.

Reply:
@marefr @jackw I have changed the text a bit, please read through it again https://github.com/grafana/grafana/pull/50356/commits/0339020b6b7c9377d7652343df8ba0f3e53eb6c3